iina/MainWindowController.swift

```diff
@@ -1088,9 +1088,7 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
     let percantage = (pos.second / duration.second) * 100
     leftLabel.stringValue = pos.stringRepresentation
     touchBarCurrentPosLabel?.stringValue = pos.stringRepresentation
-    if withDuration {
-      rightLabel.stringValue = duration.stringRepresentation
-    }
+    rightLabel.updateText(with: duration, given: pos)
```

Comment:
Seems like when the control bar is shown on mouse hover, withDuration is set to false. This means that, when the mode is set to remaining time, if we keep the check, the label will never be updated. The other alternative would be to set withDuration to true when the control bar is shown but I think that would amount to removing the check (which is what I did). I might have missed something but I actually don't quite understand why the label is only updated when withDuration is set to true.